tvtk/vtk_parser.py

```diff
@@ -653,12 +1085,71 @@ def _find_get_set_methods(self, klass, methods):
                         # This class breaks standard VTK conventions.
                         gsm[key] = (3, (1, 3))
                         continue
-                    elif (klass_name == 'vtkContextMouseEvent' and
+                    elif (klass_name in ('vtkContextMouseEvent',
+                                         'vtkInterpolateDataSetAttributes',
+                                         'vtkInteractorStyleSwitchBase') and
                           key == 'Interactor'):
                         # On VTK 8.1.0 this segfaults when uninitialized.
                         default = None
+                    # On VTK 9.0.0 these segfault when uninitialized
```

Comment:
These are much worse, when you query these classes after instantiation you get a segfault

Reply:
Some of these look suspicious. Can you reproduce these crashes in a c++ test?

Reply:
Ah, nevermind. They are easy enough to trigger in python!

Reply:
Indeed here is what happens for me with a minimal equivalent
```
>>> import vtk
>>> vtk.vtkJavaScriptDataWriter
<class 'vtkmodules.vtkIOCore.vtkJavaScriptDataWriter'>
>>> x = vtk.vtkJavaScriptDataWriter()
>>> del x
Segmentation fault
```

Reply:
... actually that example is for `vtkJavaScriptDataWriter`. It's possible that some of the below are due to memory corruptions due just to `vtkJavaScriptDataWriter` breaking things (I wrote these skips before I skipped that class entirely), as I do seem to be able to get the attributes of some of these manually with:
```
>>> import vtk
>>> x = vtk.vtkJSONDataSetWriter()
>>> x.GetFileName()
>>>
```

Reply:
It would be interesting to see what this list becomes once https://gitlab.kitware.com/vtk/vtk/-/merge_requests/6728 is applied.